core/domain/suggestion_services.py

```diff
@@ -892,16 +892,14 @@ def get_info_about_suggestions_waiting_too_long_for_review():
         submitted for review. The objects are sorted in descending order based
         on review wait time.
     """
-    suggestions_waiting_too_long_for_review = [
+    suggestions = [
```

Comment:
No, I meant keep suggestions_waiting_too_long_for_review here, but only change the dummy variable below.